tgui/packages/tgui/interfaces/VendingConsole.tsx

```diff
@@ -0,0 +1,132 @@
+import { map } from 'common/collections';
+import { useBackend, useLocalState } from '../backend';
+import { Box, ProgressBar } from '../components';
+import { Window } from '../layouts';
+
+import {
+  NanoMap,
+  NanoMapMarkerIcon,
+  NanoMapStaticPayload,
+  NanoMapTrackData,
+} from '../components/NanoMap';
+
+const pickColor = (machine: VendingObject) => {
```

Comment:
```suggestion
const pickColor = (machine: VendingObject): string => {
```
это если выпендриваться и указывать тип возвращаемого объедка